ini/features.ini

```diff
@@ -258,6 +258,7 @@ HAS_MEATPACK                           = build_src_filter=+<src/feature/meatpack
 MIXING_EXTRUDER                        = build_src_filter=+<src/feature/mixing.cpp> +<src/gcode/feature/mixing/M163-M165.cpp>
 HAS_PRUSA_MMU1                         = build_src_filter=+<src/feature/mmu/mmu.cpp>
 HAS_PRUSA_MMU2                         = build_src_filter=+<src/feature/mmu/mmu2.cpp> +<src/gcode/feature/prusa_MMU2>
+HAS_PRUSA_MMU3                         = build_src_filter=+<src/feature/mmu3/mmu2.cpp> +<src/feature/mmu3/messages.cpp> +<src/feature/mmu3/mmu2_crc.cpp> +<src/feature/mmu3/mmu2_error_converter.cpp> +<src/feature/mmu3/mmu2_fsensor.cpp> +<src/feature/mmu3/mmu2_log.cpp> +<src/feature/mmu3/mmu2_marlin1.cpp> +<src/feature/mmu3/mmu2_power.cpp> +<src/feature/mmu3/mmu2_progress_converter.cpp> +<src/feature/mmu3/mmu2_protocol_logic.cpp> +<src/feature/mmu3/mmu2_protocol.cpp> +<src/feature/mmu3/mmu2_reporting.cpp> +<src/feature/mmu3/sound.cpp> +<src/feature/mmu3/SpoolJoin.cpp> +<src/feature/mmu3/ultralcd.cpp> +<src/gcode/feature/prusa_MMU2>
 PASSWORD_FEATURE                       = build_src_filter=+<src/feature/password> +<src/gcode/feature/password>
```

Comment:
If you only want to enable all the `.cpp` files in the `src/feature/mmu3` folder then you only need `+<src/feature/mmu3>` with our latest build scripts.